_studio/mfx_lib/decode/vc1/src/mfx_vc1_decode.cpp

```diff
@@ -49,6 +49,12 @@ static void SetFrameType(const UMC::VideoData &mediaData, mfxFrameSurface1 &surf
         case UMC::I_PICTURE:
             extFrameInfo->FrameType = MFX_FRAMETYPE_I;
             break;
+        case UMC::P_PICTURE:
+            extFrameInfo->FrameType = MFX_FRAMETYPE_P;
+            break;
+        case UMC::B_PICTURE:
```

Comment:
VC1 has B and BI type.  Probably the issue you commented at https://github.com/Intel-Media-SDK/MediaSDK/issues/333#issuecomment-468352510 is caused BI type is not handled here.

Reply:
BI-Frames handled as B:
`case  VC1_B_FRAME:
 case  VC1_BI_FRAME:
     out_data->SetFrameType(B_PICTURE);
     break;`
umc_vc1_video_decoder_hw.h: line 169

Reply:
Hm, I see. So what is the frame_tpye which is still reported as unknown?

Reply:
VC1_SKIPPED_FRAME reported as unknown, should be fixed in the latest update this PR